distributed/scheduler.py

```diff
@@ -422,6 +426,17 @@ def start(self, addr_or_port=8786, start_queues=True):
             for k, v in self.services.items():
                 logger.info("%11s at: %25s", k, '%s:%d' % (self.ip, v.port))
 
+        if self.scheduler_file:
+            with ignoring(ImportError):
```

Comment:
ImportError has been removed.  I'm lightly in favor of leaving this code here.  I think that there are two competing objectives here:

1.  Pull out blocks of self-related code from high level control flow.  We don't want readers to have to sift through the details of scheduler files when looking at basic instantiation code.
2.  Reduce the amount of indirection that has to happen when looking through a codebase.  We don't want readers to have to navigate to many different parts of the file to understand how something works.

I tend to favor the second objective in cases like these, particularly because these are only a few lines, there is no repetition of this code elsewhere, and the code is easy the skip over when reading (it isn't highly connected to surrounding lines.  However, my preference here is only slight.  I'm happy to change this if your preference is stronger.